src/app/containers/CpsRecommendations/index.jsx

```diff
@@ -0,0 +1,203 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, number, string } from 'prop-types';
+import styled from 'styled-components';
+import path from 'ramda/src/path';
+import SectionLabel from '@bbc/psammead-section-label';
+import StoryPromo, { Headline, Link } from '@bbc/psammead-story-promo';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { C_LUNAR, C_EBON, C_GHOST } from '@bbc/psammead-styles/colours';
+import Image from '@bbc/psammead-image';
+import { getDoublePica } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import {
+  GEL_SPACING,
+  GEL_SPACING_HLF,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/spacings';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+import useToggle from '#hooks/useToggle';
+import CpsOnwardJourney from '../CpsOnwardJourney';
+import Grid from '../../components/Grid';
+import data from './recommendations.ltr.json';
+
+const GEL_SPACING_DBL_MINUS_HLF = '0.75rem';
+
+const StoryPromoWrapper = styled(StoryPromo)`
+  display: grid;
+  margin: ${GEL_SPACING_HLF} ${GEL_SPACING_DBL};
+  background-color: ${C_GHOST};
+`;
+
+const CustomSectionLabel = styled(SectionLabel).attrs(props => ({
+  'aria-labelledby': props.labelId,
+}))`
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSansRegular(service)};
+  color: ${C_EBON};
+  margin-top: 0;
+  background-color: ${C_LUNAR};
+`;
+
+const SectionLabelWrapper = styled.div`
+  padding: ${GEL_SPACING_DBL};
+`;
+
+const WsojWrapper = styled.div`
```

Comment:
Maybe change to `RecommendationsWrapper`

Reply:
It looks like this has been addressed.